opea_release_data.md

```diff
@@ -0,0 +1,49 @@
+# OPEA Release Data
+
+This page shows the benchmark data of GenAIExamples. More data for different examples will be submitted in the future release.
+
+## ChatQnA
+
+| **Docker Images for Test**                            |
+| ----------------------------------------------------- |
+| opea/embedding-tei:v0.9                               |
+| ghcr.io/huggingface/text-embeddings-inference:cpu-1.5 |
+| opea/llm-tgi:v0.9                                     |
+| ghcr.io/huggingface/tgi-gaudi:2.0.1                   |
+| opea/dataprep-redis:v0.9                              |
+| redis/redis-stack:7.2.0-v9                            |
+| opea/reranking-tei:v0.9                               |
+| opea/tei-gaudi:v0.9                                   |
+| opea/retriever-redis:v0.9                             |
+| opea/chatqna:v0.9                                     |
+
+System Summary:  
+4-nodes, 2x Intel(R) Xeon(R) Platinum 8380 CPU @ 2.30GHz, 40 cores, 270W TDP, HT On, Turbo On, NUMA 2, Integrated Accelerators Available [used]: DLB 0 [0], DSA 0 [0], IAA 0 [0], QAT 0 [0], Total Memory 1024GB (32x32GB DDR4 3200 MT/s [3200 MT/s]), BIOS ETM02, microcode 0xd0003b9, 8x Habana Labs Ltd., 4x MT28800 Family [ConnectX-5 Ex], 4x 7T INTEL SSDPF2KX076TZ, 2x 894.3G SAMSUNG MZ1L2960HCJR-00A07, Ubuntu 22.04.3 LTS, 5.15.0-92-generic. Software: WORKLOAD+VERSION, COMPILER, LIBRARIES, OTHER_SW. Test by Intel as of 08/20/24.
```

Comment:
for me the system summary is not quite clear for h/w configurations. is it standard format?

for example, what's that mean?
> 4-nodes, 2x Intel(R) Xeon(R) Platinum 8380 CPU @ 2.30GHz,

> 8x Habana Labs Ltd.

Reply:
This is generate by perfPDT system scan tool, so I think it's ok to use. I will let Padma confirm this. 
4-nodes means we use 4 node cluster, `2x Intel(R) Xeon(R) Platinum 8380 CPU @ 2.30GHz ` means 2 sockets.